crates/kubelet/src/server.rs

```diff
@@ -68,15 +68,46 @@ fn get_ping() -> Response<Body> {
 /// Implements the kubelet path /containerLogs/{namespace}/{pod}/{container}
 async fn get_container_logs<T: Provider + Sync>(
     provider: &T,
-    _req: &Request<Body>,
+    req: &Request<Body>,
 ) -> Response<Body> {
-    // TODO: extract the right data from the request.
+    // Basic validation steps
+    let path: Vec<&str> = req.uri().path().split('/').collect();
+    // Because of the leading slash, index 0 is an empty string. Index 1 is the
+    // container logs path
+    if path.len() != 5 {
+        let mut res = Response::new(Body::from(format!(
+            "Resource {} not found",
+            req.uri().path()
+        )));
+        *res.status_mut() = StatusCode::NOT_FOUND;
+        return res;
+    }
+
+    let namespace = path[2];
```

Comment:
```rust
let [_, _, namespace, pod, container] = path;
```

Reply:
Oh so that’s how it’s done 😁. Had a feeling you might know